addons/point_of_sale/static/src/app/utils/hooks.js

```diff
@@ -93,37 +92,6 @@ export function useAutoFocusToLast() {
     onPatched(autofocus);
 }
 
-export function useValidateCashInput(inputRef, startingValue) {
-    const cashInput = useRef(inputRef);
-    const localization = useService("localization");
-    const decimalPoint = localization.decimalPoint;
-    const env = useEnv();
-    function handleCashInputChange(event) {
-        const inputValue = (event.target.value || "").trim();
-
-        // Check if the current input value is a valid float
-        const inputElement = event.target.closest(".input-container") || event.target;
-        if (!env.utils.isValidFloat(inputValue)) {
-            inputElement.classList.add("is-invalid");
-            event.target.classList.add("invalid-cash-input");
-        } else {
-            inputElement.classList.remove("is-invalid");
-            event.target.classList.remove("invalid-cash-input");
-        }
-    }
-    onMounted(() => {
-        if (cashInput.el) {
-            cashInput.el.value = (startingValue || 0).toString().replace(".", decimalPoint);
-            cashInput.el.addEventListener("input", handleCashInputChange);
-        }
-    });
-    onWillUnmount(() => {
-        if (cashInput.el) {
-            cashInput.el.removeEventListener("input", handleCashInputChange);
-        }
-    });
-}
-
```

Comment:
I remember that that purpose of this is to make sure that user input matches the localization. For example, the user's odoo setting is that his decimal separator is ",". In that case, it doesn't matter whether his computer is setup to be in US where decimal separator is ".", when he inputs "2100,50" it should be interpreted as "2100.50". I don't want to remove that. Also, this hook fixes some number inputting in safari.

Don't just remove it, find a way to include it in this refactor. This is important.